TombEngine/Game/Lara/PlayerContextData.h

```diff
@@ -8,9 +8,9 @@ namespace TEN::Entities::Player
 		int	  LowerFloorBound = 0;
 		int	  UpperFloorBound = 0;
 
-		bool TestSlipperySlopeBelow = true;
-		bool TestSlipperySlopeAbove = true;
-		bool TestDeathFloor			= true;
+		bool TestIllegalFloorBelow = true;
```

Comment:
Why slidable slope is now called illegal floor? In TE and TRLE terminology, "illegal slope" means a slidable slope which can get player into softlock or buggy motion. I suggest to avoid using word "illegal" in this context.